### PR TITLE
cob-403-fix-Invoice-request-and-response

### PR DESCRIPTION
**Pull request by ffeitosa**
- [x] Fix invoice Request
- [x] Fix invoice Response
- [x] Fix tests

### CODE REVIEW

File: src/main/java/br/com/moip/resource/Invoice.java [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
 
 import java.util.Date;
+import java.util.List;
 
 public class Invoice {
-    private String id, description;
-    private int invoiceAmount;
-    private Date createdAt, updatedAt;
 
+    private String id;
```
Comment (SuelenGC) [inline src/main/java/br/com/moip/resource/Invoice.java]: Here comes the `external_id`? If yes, what do you think in change the name to `external_id`?

File: src/test/java/br/com/moip/api/InvoiceAPITest.java [hunk 2]
```diff
@@ -53,15 +51,18 @@ public void testCreate() {
                                                 new int[]{6, 9})
                                         .supressShippingAddress(true)));
 
-        assertEquals("INV-7761BDB06412", invoiceCreated.getId());
+        assertEquals("INV-B36128A22351", invoiceCreated.getId());
+        assertEquals(Integer.valueOf(1000), invoiceCreated.getAmount().getTotal());
+        assertEquals("My Invoice Product", invoiceCreated.getItems().get(0).getProduct());
```
Comment (SuelenGC) [inline src/test/java/br/com/moip/api/InvoiceAPITest.java]: The product and detail asserts are ok? Because you create invoice with a different text...

**Comment by SuelenGC** [comment]
👍

File: src/test/java/br/com/moip/api/InvoiceAPITest.java (hunk 2 above)
Comment (ffeitosa) [inline src/test/java/br/com/moip/api/InvoiceAPITest.java]: Fixed

**Pull request merged**